src/driver/amdxdna/amdxdna_drv.c

```diff
@@ -55,6 +56,12 @@ static int amdxdna_drm_open(struct drm_device *ddev, struct drm_file *filp)
 	struct amdxdna_client *client;
 	int ret;
 
+	ret = pm_runtime_resume_and_get(ddev->dev);
+	if (ret < 0) {
```

Comment:
```suggestion
	if (ret) {
```
Although this function is documented as it will only return 0 or negative error code. But, when ret is somehow positive, the suggestion is a stronger code and good for maintenance.